lib/bundle/commands/cleanup.rb

```diff
@@ -41,6 +43,12 @@ def run(global: false, file: nil, force: false, zap: false)
             Kernel.system "code", "--uninstall-extension", extension
           end
 
+          # TODO: See how we can get around the sudo requirement because of the way e.g. basictex installs itself
+          # See also https://tug.org/texlive/doc/tlmgr.html#USER-MODE
+          tlmgr_packages.each do |package|
+            Kernel.system "sudo", "tlmgr", "remove", package
```

Comment:
I'm afraid there's no way we're going to accept this if `sudo` is needed here, sorry.